src/panoptes/pocs/focuser/birger.py

```diff
@@ -63,19 +62,16 @@ class Focuser(AbstractFocuser):
     # Class variable to cache the device node scanning results
     _birger_nodes = None
```

Comment:
```suggestion
```

Don't need this as `_adaptor_nodes = None` in `serial.py` serves this purpose more generically.

Reply:
Change all use of `_birger_nodes` to `_adaptor_nodes`.